### PR TITLE
Ports ID modifications from OnyxBay

### PR DESCRIPTION
**Pull request by quardbreak**
Allows people with id modification access change fields and recover data from records.
closes https://github.com/NebulaSS13/NebulaFeatureRequests/issues/37

### CODE REVIEW

File: code/modules/modular_computers/file_system/programs/command/card.dm [hunk 1]
```diff
@@ -101,6 +108,24 @@
 /datum/nano_module/program/card_mod/proc/get_accesses(var/is_centcom = 0)
 	return null
 
+/datum/computer_file/program/card_mod/proc/get_photo(mob/user)
+	if(istype(user.get_active_hand(), /obj/item/photo))
+		var/obj/item/photo/photo = user.get_active_hand()
+		return photo.img
+
+	if(istype(user, /mob/living/silicon))
+		var/mob/living/silicon/tempAI = usr
```
Comment (comma) [inline code/modules/modular_computers/file_system/programs/command/card.dm]: user not usr

**Comment by MistakeNot4892** [comment]
Closing as eckff is AWOL, feel free to reopen.

**Comment by Lohikar** [comment]
Reopened on request of author.

**Review by MistakeNot4892 — DISMISSED** [review]

File: code/modules/modular_computers/file_system/programs/command/card.dm [hunk 2]
```diff
@@ -193,6 +221,57 @@
...
+				else if(href_list["fingerprint_hash"])
+					var/sug_fingerprint_hash = sanitize(input("Enter fingerprint hash.", "Fingerprint hash", id_card.fingerprint_hash), 32)
+					if(hash_check.Find(sug_fingerprint_hash))
+						id_card.fingerprint_hash = sug_fingerprint_hash
+				else if(href_list["dna_hash"])
+					var/sug_dna_hash = sanitize(input("Enter DNA hash.", "DNA hash", id_card.dna_hash), 32)
+					if(hash_check.Find(sug_dna_hash))
+						id_card.dna_hash = sug_dna_hash
+				else if(href_list["blood_type"])
+					var/sug_blood_type = input("Select blood type.", "Blood type") as null|anything in GLOB.blood_types
```
Comment (afterthought2) [inline code/modules/modular_computers/file_system/programs/command/card.dm:241]: Need to do a `CanUseTopic`-type check after every `input` call in this whole proc.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**